msgraph-mail/go/utilities/models/attachment.go

```
@@ -32,7 +22,14 @@ func CreateAttachmentFromDiscriminatorValue(parseNode i878a80d2330e89d26896388a3
 }
 // GetContentType gets the contentType property value. The MIME type.
 func (m *Attachment) GetContentType()(*string) {
-    return m.contentType
+    val , err := m.GetBackingStore().Get("contentType")
+    if val != nil {
+        return val.(*string)
+    }
+    if err != nil {
+        panic(err)
+    }
```

I think the two should be inverted, otherwise we're swallowing potential errors